lib/clients/OpenstackClient.py

```diff
@@ -16,15 +16,14 @@ class OpenstackClient(BaseClient):
     def __init__(self, operation_name, configuration, directory_persistent, directory_work_list, poll_delay_time,
                  poll_maximum_time):
         super(OpenstackClient, self).__init__(operation_name, configuration, directory_persistent, directory_work_list,
-                                              poll_delay_time, poll_maximum_time)
-        self.__keystoneCredentials = {
-            'username': configuration['username'],
-            'password': configuration['password'],
-            'auth_url': configuration['auth_url'],
-            'user_domain_name': configuration['user_domain_name'],
-            'project_id': configuration['tenant_id'],
-            'project_name': configuration['tenant_name']
-        }
+                                              poll_delay_time, poll_maximum_time)      
+        if not 'credhub_url' in configuration:
+            self.__setCredentials(configuration)
+        else:
+            self.logger.info('fetching creds from credhub')
+            credentials = self._get_credentials_from_credhub(configuration)
```

Comment:
Same as above. What happens if there is an error in getting these credentials? Do we need to raise exception explicitly?